test/built-ins/Temporal/Duration/prototype/round/roundingincrement-non-integer.js

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2023 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.duration.prototype.round
+description: Rounding for roundingIncrement option
+info: |
+    sec-temporal-totemporalroundingincrement:
+      3. Let _integerIncrement_ be truncate(ℝ(_increment_)).
```

Comment:
In this specific case I'm happy to change the formatting and quote the whole ToTemporalRoundingIncrement since I can just copy-paste-automate that. But in general I think extra time put into these is likely to be excessive compared to their benefit.